var/spack/repos/builtin/packages/gslib/package.py

```diff
@@ -0,0 +1,64 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Gslib(Package):
+    """Highly scalable Gather-scatter code with AMG and XXT solvers"""
+
+    homepage = "https://github.com/gslib/gslib"
+    url      = "https://github.com/gslib/gslib"
+
+    version('v1.0.1', git='https://github.com/gslib/gslib.git',
```

Comment:
we usually don't add `v`, just do `1.0.1`.

Reply:
I am worried if the users will get confused about the versions we have [here](https://github.com/gslib/gslib/releases)
and what they have to type into `spack`. I am thinking of a `gslib` user using
`spack` to get it installed her system. Can we just keep this as it is? Does 
adding a `v` in front of the number matter when doing relational operations like checking
if  `version A` came before or after `version B`?